### PR TITLE
add a reviewer, please

### PR DESCRIPTION
**Pull request by diogoEsteves42**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ test emails: felipe@example.com
 
 ## author
 by felipe & diogo
+
+## another field
```
Comment (felipegc) [inline README.md:14]: which field?

**Review by felipegc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (diogoEsteves42) [reply README.md:14]: I don't know.

Comment (felipegc) [reply README.md:14]: just add something

Comment (diogoEsteves42) [reply README.md:14]: holy molly!

**Comment by diogoEsteves42** [comment]
done

File: README.md [hunk 2]
```diff
@@ -10,3 +10,6 @@ test emails: felipe@example.com
...
+another field
```
Comment (felipegc) [inline README.md:15]: change to outro field

**Review by felipegc — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (diogoEsteves42) [reply README.md:15]: changed

File: README.md [hunk 3]
```diff
@@ -10,3 +10,6 @@ test emails: felipe@example.com
 
 ## author
 by felipe & diogo
+
+## another field
+outro field
```
Comment (felipegc) [inline README.md:15]: delete this line

**Review by felipegc — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (diogoEsteves42) [reply README.md:15]: done

File: README.md (hunk 1 above)
Comment (felipegc) [inline README.md:14]: add back a new field

**Review by felipegc — CHANGES_REQUESTED** [review]

**Review by felipegc — APPROVED** [review]

**Pull request merged**